CMakeLists.txt

```diff
@@ -171,9 +200,15 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 	CXX_EXTENSIONS ON
 )
 
-install(TARGETS ${PROJECT_NAME}
-	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+set_target_properties(modelcompiler PROPERTIES
```

Comment:
You can specify both ```${PROJECT_NAME}``` and ```modelcompiler``` in set_target_properties(), to avoid duplicating code.